app/src/pages/portal/chart/DeleteChart.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="">
+    <div v-if="loading" class="section_loader">
+      <spinner :loading="true" :text="loadingMessage" />
+    </div>
+
+    <div v-else class="">
+      <div class=" viz-u-mgup-sm utility-margin md-theme-default">
+        <div class="md-card-header contactus_radios md-card-header-flex">
+          <div class="md-card-header-text">
+            <div class="md-body-1">This task requires super admin priviledges, deploying unapproved changes will require a rollback</div>
```

Comment:
Is this the correct description for this one?

Reply:
We can definitely reword it. Any suggestion?

Reply:
The wording is fine, I was just confused because it's the same one used for the "Deploy" page and wasn't sure if that was intentional